docs-2.0/3.ngql-guide/11.edge-statements/3.upsert-edge.md

```diff
@@ -0,0 +1,42 @@
+# UPSERT EDGE
+
+```ngql
+UPSERT EDGE <src_vid> -> <dst_vid> [@rank] OF <edge_type> SET <update_properties> [WHEN <condition>] [YIELD <properties>]
+```
+
+- `update_properties` is the properties of the edge to be updated. For example, `e1.col1 = $^.e1.col2 + 1` means to update `e1.col1` to `e1.col2+1`.
+
+    > **NOTE:**  `$^` indicates the edge to be updated.
+
+- `condition` is some constraints. Only when the `condition` is met, `UPSERT` is executed successfully. `condition` supports expression operations.
+- `properties` specify the properties to be returned, `YIELD` returns the latest updated values.
```

Comment:
specifies. 

through the article, we treat all the parameters as a whole (singular noun), so keeping them consistent is recommended.